themes/be.software.designers/theme.json

```diff
@@ -7,7 +7,7 @@
     "forDesigners": true,
     "externalResources": {
         "stylesheets": [
-            "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+            "/custom-assets/font-awesome-4.7.0/css/font-awesome.min.css"
```

Comment:
Do we want to target the public folder instead as custom-assets is in there? i.e.
`"../../public/custom-assets/font-awesome-4.7.0/css/font-awesome.min.css"`